### PR TITLE
Paradoxical Vuvuzela

### PR DESCRIPTION
**Pull request by ghost**
wheezl paid me to do this. waiting for sprites.

:cl:
add: Added the Paradoxical Vuvuzela. It plays a very horrible sound.
/:cl:

https://www.youtube.com/watch?v=dbT7D2lIN9c

### CODE REVIEW

**Comment by karmaisblackandbluepilled** [comment]
This plays a literal 23 minutes of a sound file. It's 20 mb long.
What the fuck did gms have again? -3? -2? However much I can give, minus.

**Comment by KingTerrytheTerrible** [comment]
wtf i stop 

> 
> 
> This plays a literal 23 minutes of a sound file. It's 20 mb long.
> What the fuck did gms have again? -3? -2? However much I can give, minus.

stopsound

**Comment by karmaisblackandbluepilled** [comment]
don't care

**Comment by carlopault** [comment]
is that **steamp0rt**? +1

**Comment by wheezl** [comment]
+4

File: hippiestation/code/game/objects/items/paradox_vuvuzela.dm [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	name = "paradoxical vuvuzela"
+	desc = "Only a clown of a mad scientist would think this is a good instrument!"
+	var/sound_to_play = 'hippiestation/sound/misc/paradox.ogg'
+	var/cooldown = 15 SECONDS
+	var/next_play = 0
+
+/obj/item/paradoxical_vuvuzela/attack_self(mob/user)
+	. = ..()
+	if(next_play > world.time)
+		to_chat(user, "<span class='notice'>You need to wait [DisplayTimeText(next_play - world.time)] before you can use \the [src] again.</span>")
```
Comment (Neotwr) [inline hippiestation/code/game/objects/items/paradox_vuvuzela.dm:16]: remove `\the`, it isn't necessary

Comment (ghost) [reply hippiestation/code/game/objects/items/paradox_vuvuzela.dm:16]: it's a code styling choice

**Comment by FantasticFwoosh** [comment]
PTW

**Comment by ghost** [comment]
Tested and ready.

**Comment by AnthonyJNgo** [comment]
+3

**Review by yoyobatty — APPROVED** [review]

**Pull request merged**